evm-contracts/src/v0.6/dev/ControllerInterface.sol

```diff
@@ -0,0 +1,5 @@
+pragma solidity ^0.6.0;
+
+interface ControllerInterface {
+  function access(address user, bytes calldata data) external view returns (bool);
```

Comment:
Doesn't this need to be public in order for checkAccess to call it in Controller.sol?

Reply:
All functions declared in interfaces should be marked as external.